### PR TITLE
docs: How to add worker pool to separate facility

### PR DESCRIPTION
**Pull request by surajssd**
This commit adds a doc that shows how to add a new worker pool to a
separate facility than the control plane and other worker pools.

### CODE REVIEW

**Comment by surajssd** [comment]
Dependent on https://github.com/kinvolk/lokomotive/pull/1359 to be merged.

File: docs/how-to-guides/worker-pool-separate-facility.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Step 1: Enable "Backend Transfer"
+
+Go to the Equinix Metal console of your project and enable "Backend Transfer" on it, follow [this
+document](https://metal.equinix.com/developers/docs/networking/features/#backend-transfer) for
+detailed information.
+
+### Step 2: Add private CIDR of the new facility
+
+Go to the **Equinix Metal console** > **IPs & Networks** > **IPs**.
```
Comment (iaguis) [inline docs/how-to-guides/worker-pool-separate-facility.md:32]:
```suggestion
Go to **Equinix Metal console** > **IPs & Networks** > **IPs**.
```

File: docs/how-to-guides/worker-pool-separate-facility.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+`10.xx.xx.xx/25` range.
+
+Open your Lokomotive cluster's `lokocfg` file and add it to the existing `node_private_cidrs` list.
+
+```tf
+node_private_cidrs = ["10.10.10.128/25", "10.xx.xx.xx/25"]
+```
+### Step 3: Add worker pool with a different facility
+
+Add the following snippet to your existing `lokocfg` file, under `cluster "packet"` section:
```
Comment (iaguis) [inline docs/how-to-guides/worker-pool-separate-facility.md:44]:
```suggestion
Add the following snippet to your existing `lokocfg` file, under the `cluster "packet"` section:
```

**Review by iaguis — CHANGES_REQUESTED** [review]
Nits

**Review by iaguis — APPROVED** [review]
lgtm

**Pull request merged**